### ISSUE TITLE
PHP log error caused by new account request

### ISSUE THREAD

**Issue by ahelsing** [GENI-NSF/geni-ar#81]
When i make a new account request using 1.2.1, the apache error log gets:

```
[Wed Feb 05 11:27:40 2014] [error] [client 128.89.254.73] PHP Notice:  Undefined variable: AR_EMAIL_HEADERS in /usr/share/geni-ar/www/handlerequest.php on line 381, referer: https://shib-idp1.gpolab.bbn.com/geni/request.html
```

It's not yet obvious to me why this is or what the other implications are, but i'll keep an eye open.

 _Imported from trac ticket [#81](http://trac.gpolab.bbn.com/idp-tools/ticket/81),  created by chaos on 02-05-2014 at 11:31, last modified: 02-05-2014 at 15:37_

**Comment by ahelsing** [comment]
At a glance, it looks like this variable is being set in:

```
/usr/share/geni-ar/lib/php/ar_constants.php
```

and is the one which tells the e-mail to set certain headers including the reply-to.  Indeed, there is no Reply-To on the copy of the e-mail which was sent to the audit address, whereas in 1.2, the Reply-To was set to (hardcoded) portal-help.

Is it possible that handlerequest.php does not require ar_constants.php, and should?

_Trac comment by chaos on 02-05-2014 at 11:35_

**Comment by ahelsing** [comment]
Yes, looks like that is the problem, sorry!  I can make a release 1.2.2 branch and fix that

_Trac comment by phelinek on 02-05-2014 at 12:38_

**Comment by ahelsing** [comment]
Sounds good.  I think this one is a probable-blocker with a easy fix, so it merits a 1.2.2.

_Trac comment by chaos on 02-05-2014 at 12:40_

**Comment by ahelsing** [comment]
Ok I fixed handle-request and pushed the new branch

_Trac comment by phelinek on 02-05-2014 at 13:07_

**Comment by ahelsing** [comment]
Verified that this is fixed in 1.2.2.

_Trac comment by chaos on 02-05-2014 at 15:37_
